saged/predict_tissue.py

```diff
@@ -0,0 +1,172 @@
+"""
+This benchmark compares the performance of different models in
+predicting tissue based on gene expression
+"""
+import argparse
+
+import sklearn.metrics
+import yaml
+
+from saged import utils, datasets, models
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('dataset_config',
+                        help='The yaml formatted dataset configuration file. For more information '
+                             'about this file read the comments in the example_dataset.yml file')
+    parser.add_argument('supervised_config',
+                        help='The yaml formatted model configuration file. For more information '
+                             'about this file read the comments in the example_model.yml file')
+    parser.add_argument('out_file',
+                        help='The file to save the results to')
+    parser.add_argument('--tissue1',
+                        help='The first tissue to be predicted from the data',
+                        default='Blood')
```

Comment:
Do you have a list of the tissue options somewhere? You could use that here as the `choices` argument if you have it.

Reply:
Ooh good idea, will add it